src/pages/index.js

```diff
@@ -148,7 +148,7 @@ const renderCard = (item) => {
   );
   const cardElement = addNewCard.getCard();
   cardSection.addItem(cardElement);
-}; */
+};
```

Comment:
`card._cardId`


Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a `private` method which is used only inside the class.


```js

card.cardId
```